benchmarks/movielens.ipynb

```diff
@@ -0,0 +1,942 @@
+       "      <td>0:00:00.037825</td>\n",
+       "      <td>0.961177</td>\n",
+       "      <td>0.749250</td>\n",
+       "      <td>0.261103</td>\n",
+       "      <td>0.257623</td>\n",
+       "      <td>0:00:00.066403</td>\n",
+       "      <td>0.005953</td>\n",
+       "      <td>0.053793</td>\n",
+       "      <td>0.054825</td>\n",
+       "      <td>0.018347</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>100k</td>\n",
+       "      <td>svd</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:09.535290</td>\n",
+       "      <td>0:00:02.710756</td>\n",
+       "      <td>0.945304</td>\n",
+       "      <td>0.744339</td>\n",
+       "      <td>0.281940</td>\n",
+       "      <td>0.281986</td>\n",
+       "      <td>0:00:13.131252</td>\n",
+       "      <td>0.015410</td>\n",
+       "      <td>0.111125</td>\n",
+       "      <td>0.099152</td>\n",
+       "      <td>0.034326</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>100k</td>\n",
+       "      <td>sar</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:00.215702</td>\n",
+       "      <td>None</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>0:00:00.101380</td>\n",
+       "      <td>0.003903</td>\n",
+       "      <td>0.033111</td>\n",
+       "      <td>0.040827</td>\n",
+       "      <td>0.024010</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>100k</td>\n",
+       "      <td>ncf</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:20.404797</td>\n",
+       "      <td>None</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>0:00:02.681851</td>\n",
+       "      <td>0.091839</td>\n",
+       "      <td>0.355767</td>\n",
+       "      <td>0.309544</td>\n",
+       "      <td>0.161812</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>100k</td>\n",
+       "      <td>fastai</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:20.793080</td>\n",
+       "      <td>0:00:00.040010</td>\n",
+       "      <td>0.903492</td>\n",
+       "      <td>0.714408</td>\n",
+       "      <td>0.344057</td>\n",
+       "      <td>0.344997</td>\n",
+       "      <td>0:00:03.051865</td>\n",
+       "      <td>0.027357</td>\n",
+       "      <td>0.158870</td>\n",
+       "      <td>0.140403</td>\n",
+       "      <td>0.056657</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>1m</td>\n",
+       "      <td>als</td>\n",
+       "      <td>10</td>\n",
+       "      <td>0:00:03.275541</td>\n",
+       "      <td>0:00:00.013025</td>\n",
```

Comment:
Just double check if als' rating and ranking time were correctly measured. Isn't the transform lazy operation? 

Also would be better to use more 'human-friendly' time format, maybe 'secs'?

Reply:
changed in the next commit